examples/window_size.rs

```diff
@@ -0,0 +1,80 @@
+use std::{error::Error, io};
```

Comment:
It could be worthwhile adding some doc comments to help a reader understand what this example shows.

Reply:
I removed this in the end, as it's not a valuable demo/example.